microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -466,6 +466,10 @@ If you previously saved payment information to your Microsoft account, it is als
 
```

Comment:
Suggest changing the two h3's in this article to h4's to make page structure perceptible and clear:
Change:
```
### Microsoft News
```
to:
```
#### Microsoft News
```

Change:
```
### Office 365
```
to:
```
#### Office 365
```